src/redis-cli.c

```diff
@@ -2950,6 +2952,9 @@ static int parseOptions(int argc, char **argv) {
             config.test_hint = argv[++i];
         } else if (!strcmp(argv[i],"--test_hint_file") && !lastarg) {
             config.test_hint_file = argv[++i];
+        } else if (!strcmp(argv[i], "--test-panic") && !lastarg) {
+            config.test_panic = 1;
+            config.test_panic_msg = argv[++i];
```

Comment:
why no just using `debug panic`?

Reply:
there two are for redis-cli, i think we should verify the crash log from redis-server.

Reply:
This redisassert.c modification only affects reidis-cli and redis-benchmark, and redis-server uses _serverPanic defined in the debug.c file, so only the redis-cli verification test is performed this time

Reply:
However, redis-cli does not have similar test modules as redis-server

Reply:
you're right, let me think again.

Reply:
@cyy-tag i think we can skip the test(revert the last commit), and test it manually. thanks.

Reply:
@sundb agree, I have revert and tested